bin/compile

```diff
@@ -274,6 +276,16 @@ sub_env "$BIN_DIR/steps/geo-libs"
 # shellcheck source=bin/steps/gdal
 source "$BIN_DIR/steps/gdal"
 
+# SQLite3 support.
+# This sets up and installs sqlite3 dev headers and the sqlite3 binary but not the
+# libsqlite3-0 library since that exists on the stack image.
+# Note: This only applies to Python 2.7.15+ and Python 3.6.5+
```

Comment:
You said `Python 3.6.5+` here but the runtime build script for 3.6.5 has not been updated to install Sqlite3

Reply:
Good catch! Thanks.